checks/check_status.py

```diff
@@ -127,7 +127,8 @@ def validate_api_key(config):
         if proxy:
             request_proxy = {'https': "http://{user}:{password}@{host}:{port}".format(**proxy)}
         r = requests.get("%s/api/v1/validate" % config['dd_url'].rstrip('/'),
-            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3)
+            params={'api_key': config.get('api_key')}, proxies=request_proxy, timeout=3,
+                         verify=(not config['skip_ssl_validation']))
```

Comment:
Yeah, those are both reasonable. Will fixup probably next week after Pycon. 

Also, I noticed you're using `config.get('api_key')` when I would have expected `config['api_key']` because the check doesn't really make sense without an API key. Is that on purpose?  

If so, should I be using `config.get('skip_ssl_validation', False)` as well? I haven't actually played with the agent yet, so not sure if it is already set to `False` if not explicitly set by the user.